src/meta/src/hummock/compaction/compaction_config.rs

```diff
@@ -74,7 +73,9 @@ impl CompactionConfigBuilder {
                 //    level0_max_compact_file_number * target_file_size_base >
                 // max_bytes_for_level_base
                 level0_max_compact_file_number: DEFAULT_MAX_COMPACTION_FILE_COUNT,
-                level0_sub_level_compact_level_count: DEFAULT_MIN_SUB_LEVEL_COMPACT_LEVEL_COUNT,
+                level0_sub_level_compact_level_count: (DEFAULT_MAX_BYTES_FOR_LEVEL_BASE
```

Comment:
1. we cloud remove the `DEFAULT_MIN_SUB_LEVEL_COMPACT_LEVEL_COUNT` ?
2. if we  `DEFAULT_MIN_COMPACTION_BYTES` (sub_level_max_compaction_bytes), I think we need to check the validity of the configuration  to avoid compaction exceptions caused by misconfiguration

Reply:
I would remove it in an independent PR